protocol/0069-VCBS-validators_chosen_by_stake.md

```diff
@@ -328,7 +328,7 @@ See [limited network life spec](./0073-LIMN-limited_network_life.md).
 9. Swap due to better score (<a name="0069-VCBS-042" href="#0069-VCBS-042">0069-VCBS-042</a>):
   * Setup a network with 5 Tendermint validators and self-delegate 1000 tokens to each of them. 
   * Announce a new node at the beginning of the epoch, self-delegate to them a total that is 10000 tokens. 
-  * At the beginning of the next epoch the new validator should have ranking score *equal* to all of the Tendermint validators so it doesn’t get promoted. 
+  * At the beginning of the next epoch the new validator should have ranking score *equal or lower* to all of the Tendermint validators so it doesn’t get promoted. 
```

Comment:
This makes no sense. It's the tendermint validators that never quite have a score of one, so in this step the new node will almost always have a higher score and so will get promoted.